framework/doc/content/source/meshgenerators/UniqueExtraIDMeshGenerator.md

```diff
@@ -0,0 +1,28 @@
+# UniqueExtraIDMeshGenerator
+
+!syntax description /Mesh/UniqueExtraIDMeshGenerator
+
+
+## Overview
+
+
+The `UniqueExtraIDMeshGenerator` object applies a new set of extra IDs determined by parsing existing extra IDs.
+First, the exisiting extra IDs of each element are parsed to find a set of unique combinations of their ID values.
+Then, a new ID value are determined to individual unique combiniation and assign it to the corresponding element.
+
```

Comment:
Add: The new ID is embedded in all the existing IDs, i.e. elements with a unique new ID value have a unique value for all existing IDs.